contributors.md

```diff
@@ -9,6 +9,12 @@ Includes all git commit authors. Aliases are GitHub user names.
 * Tulio Paschoalin Leao (tupaschoal)
 * Olivier Wervers (Oli414)
 * Matthias Moninger (ZehMatt)
+* Karst van Galen Last (AuraSpecs) - Lead OpenSoundEffects & OpenMusic
+* Michael Bernardi (mrmbernardi)
+
+## Support team
+* Aaron van Geffen (AaronVanGeffen)
```

Comment:
Thinking about this some more, I prefer to just be listed as a former developer. Thanks.